bin/miniplayer

```diff
@@ -499,8 +502,13 @@ class Player:
         A function that fetches the album art and saves
         it to self.album_art_loc
         """
+
         try:
-            albumart_data = self.client.albumart(song_file)
+            if MPDMETHOD == "albumart":
```

Comment:
Here, since embedded album art is often more accurate, you could implement something like
```python
albumart_data = self.client.readpicture(song_file)

if not albumart_data: # No embedded art
   # Try to read album art with client.albumart()
   # Handle CommandError if that fails
```
This would eliminate the need for additional config!